public_html/layouts/basic/modules/Settings/WebserviceUsers/resources/List.js

```diff
@@ -82,11 +82,26 @@ Settings_Vtiger_List_Js('Settings_WebserviceUsers_List_Js', {}, {
 		);
 		return aDeferred.promise();
 	},
+	/**
+	 * Register buttons to copy password
+	 */
+	registerClipBoard: function () {
+		new Clipboard('.copyPassword', {
```

Comment:
This password functionality should be universal and added in a file https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/public_html/layouts/resources/Fields.js